### DISCUSSION TITLE
How to output Dockerfile without actual build?

### DISCUSSION THREAD

**Discussion by MondayCha** [tensorchord/envd#1933] [Q&A] [answered]
Hi there! 👋

I'm interested in using envd but have a specific use case where I'd like to see the generated Dockerfile content without actually performing the build. This would be helpful for integrating with existing CI/CD pipelines that require Dockerfile.

Currently, I'm not sure if there's a way to achieve this. I've looked through the documentation but couldn't find any related flags or options.

Questions:
- Is there an existing way to output the Dockerfile content without building (might `debug`)?
- If not, would it be possible to add a `--dry-run` flag to the build command?

Example of what I'm looking for:

```bash
# Something like this
envd build --dry-run
```

**Comment by kemingy** [comment, marked as answer]
We don't have this feature for now. It's tracked at https://github.com/tensorchord/envd/issues/523.

Maybe you can ask [aseaday](https://github.com/aseaday) there about the progress.
